includes/class-custom-card.php

```diff
@@ -81,49 +81,110 @@ public function register_field_group() {
 
 			acf_add_local_field_group( [
 				'key' => 'savage_custom_card_group',
-				'title' => __( 'Card content (not ready for use)', 'savage-cards' ),
+				'title' => __( 'Card content', 'savage-cards' ),
 				'fields' => [
 					[
-						'key' => $this->field_key . '_title',
-						'label' => __( 'Card title', 'savage-cards' ),
-						'name' => 'card_title',
-						'type' => 'text',
-						'instructions' => __( 'This is the title showed on the card', 'savage-cards' ),
-						'required' => 0,
-						'maxlength' => '',
-					],
-					[
-						'key' => $this->field_key . '_tagline',
-						'label' => __( 'Tagline', 'savage-cards' ),
-						'name' => 'card_tagline',
-						'type' => 'text',
+						'key' => $this->field_key . '_link',
+						'label' => __( 'Card link', 'savage-cards' ),
+						'name' => 'card_link',
+						'type' => 'link',
 						'instructions' => '',
-						'required' => 0,
-						'default_value' => '',
-						'maxlength' => '',
+						'required' => 1,
+						'return_format' => 'array',
 					],
 					[
-						'key' => $this->field_key . '_formatted_text',
-						'label' => __( 'Text', 'savage-cards' ),
-						'name' => 'card_text',
-						'type' => 'wysiwyg',
+						'key' => $this->field_key . '_background',
+						'label' => __( 'Card background color', 'savage-cards' ),
+						'name' => 'card_background',
+						'type' => 'select',
 						'instructions' => '',
-						'required' => 0,
-						'tabs' => apply_filters( 'savage/card/custom/content/tabs', 'all' ),
-						'toolbar' => apply_filters( 'savage/card/custom/content/toolbar', 'savage_card_toolbar' ),
-						'media_upload' => 0,
+						'conditional_logic' => 0,
+						'choices' => apply_filters( 'savage/card/custom/bg_color_options', [
+							'bg_default' => __( 'Default', 'savage-cards' ),
+						] ),
+						'default_value' => [],
+						'ui' => 0,
+						'ajax' => 0,
+						'return_format' => 'value',
 					],
 					[
-						'key' => $this->field_key . '_link',
-						'label' => __( 'Link to', 'savage-cards' ),
-						'name' => 'card_link',
-						'type' => 'link',
+						'key' => $this->field_key . '_flex',
+						'label' => __( 'Card content', 'savage-cards' ),
+						'name' => 'card_content_flex',
+						'type' => 'flexible_content',
 						'instructions' => '',
-						'required' => 0,
-						'wrapper' => [
-							'width' => '50',
+						'required' => 1,
+						'layouts' => [
+							[
+								'key' => $this->field_key . '_flex_standard',
+								'name' => 'card_standard',
+								'label' => __( 'Standard', 'savage-cards' ),
+								'display' => 'block',
+								'sub_fields' => [
+									[
+										'key' => $this->field_key . '_flex_standard_img',
+										'label' => __( 'Custom card image', 'savage-cards' ),
```

Comment:
Trenger vi dette valget? Tenker kanksje at hvis man har satt featured image så vil man ha bilde.

Reply:
Nei det gjør vi ikke, keeping it simple og tar det bort. 

Phpcs-lintern hos meg reagerer ikke på at pilene ikke er alignet, verken i atom eller når jeg kjører via console. Hvor er det den regelen kommer fra? Kan være noe jeg mangler.